lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -120,6 +120,28 @@ ::mlir::LogicalResult mlir::tt::ttir::MatmulOp::verify() {
 }
 // ANCHOR_END: adding_an_op_matmul_ttir_verify
 
+::mlir::LogicalResult mlir::tt::ttir::Conv2dOp::verify() {
```

Comment:
Once we get better understanding of what tt-nn expects, let's broaden this verify. It'll be good to match all expectations of tt-nn. Likewise, to match limitations as well and have a reference point here